.github/dependabot.yml

```diff
@@ -20,6 +20,8 @@ updates:
           - "@eslint/*"
           - "yaml-eslint-parser"
           - "vue-eslint-parser"
+          - "typescript-eslint"
```

Comment:
I'm guessing this is left over from some testing, as we don't have it as a direct dependency and likely won't either as it is a dependency of neostandard so we just need to turn it on and if we need it separately we need to use their reexport to avoid problems (see discussion here https://github.com/neostandard/neostandard/issues/184).
```suggestion
```